lib/ui/layers/panels/filter.mjs

```diff
@@ -15,6 +15,7 @@ mapp.utils.merge(mapp.dictionaries, {
     layer_filter_greater_than: 'Greater than',
     layer_filter_less_than: 'Less than',
     layer_filter_set_filter: 'Set filter',
+    layer_filter_location_match: 'Location(s) match your criteria'
```

Comment:
this is going to want to wait for the Dictionaries issue #1434 as all these merges are not happening in the core anymore

Reply:
I removed this for line for now.